lib/iis_shc_issuer/qbp_client.rb

```diff
@@ -10,7 +10,7 @@ module IISSHCIssuer
   module QBPClient
     extend self
     # Query a patient's Immunization history from the IIS Sandbox
-    # @param patient_info [Hash] Patient demographic info sent from the IIS Consumer Portal
+    # @param patient_info [PatientQuery] Patient demographic info sent from the IIS Consumer Portal
     # @param credentials [Hash] (optional) User Id, Password, and Facility Id for the IIS Sandbox login
```

Comment:
I probably forgot to do this in a former PR, but would you mind updating this parameter description to say `sandbox_credentials` instead of `credentials`?